e2e/fixtures/applicant.js

```diff
@@ -1,4 +1,5 @@
 module.exports = {
+  partyType: 'Individual',
```

Comment:
If you do not use in your tests, please remove

Reply:
👍 yes boss